application-flashmessages-ui/src/main/resources/Flash/FlashNotifier.xml

```diff
@@ -153,51 +153,64 @@
       <cache>long</cache>
     </property>
     <property>
-      <code>/**
- * Confirm that the user has seen the popups
- */
-window.flashNotifierConfirm = function(params){
-  window.flashPopup.closeDialog();
-  new Ajax.Request("$doc.getURL('get')", {
-    parameters : {'ajax' : '1', 'flash': params['document']},
-    method: 'get'
+      <code>require(['jquery'], function($) {
+  var thisWindow = $(window)[0];
+  /**
+   * Initialize the flash notifier.
+   */
+  thisWindow.flashNotifier = function(params) {
+    thisWindow.flashNotifierParams = params;
+    thisWindow.flashNotifierIndex = 0;
+    thisWindow.flashNotifierNext();
+  };
+  /**
+  * Display the next popup.
+  */
+  thisWindow.flashNotifierNext = function() {
+    // If index &gt; length, stop it!
+    if(thisWindow.flashNotifierIndex &gt;= thisWindow.flashNotifierParams.length)
+      return;
```

Comment:
It's not a good idea to rely only on indentation. The next person adding a line under this if might forget to add the ``{...}``.